src/test/java/com/flagsmith/flagengine/unit/segments/SegmentModelTest.java

```diff
@@ -62,6 +62,11 @@ public Object[][] conditionTestData() {
         new Object[] {SegmentConditions.NOT_CONTAINS, "bar", "baz", true},
         new Object[] {SegmentConditions.REGEX, "foo", "[a-z]+", true},
         new Object[] {SegmentConditions.REGEX, "FOO", "[a-z]+", false},
+        new Object[] {SegmentConditions.MODULO, 2, "2|0", true},
+        new Object[] {SegmentConditions.MODULO, 3, "2|0", false},
```

Comment:
Might be worth adding a test with float condition value

Reply:
Yep, good call. I've done that.